### PR TITLE
Simple web server

### PR DESCRIPTION
**Pull request by Kinshuk1993**
- Create basic webserver that implements an unparameterized GET and a GET that takes a claim id.
- Implement GET to return full table and display it
- Implement GET to return a single claim

### CODE REVIEW

File: EventEnabledInsurance/ProjectionClaims/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,11 @@ COPY --chown=gradle:gradle src /home/gradle/src/src
 WORKDIR /home/gradle/src
 RUN gradle fatJar --no-daemon
 
-FROM openjdk:8-jre-slim
+# Use this for debug, includes curl etc:
+FROM openjdk:8
+
+# Use this for a small production image
+# TODO FROM openjdk:8-jre-slim
```
Comment (DanRoseus) [inline EventEnabledInsurance/ProjectionClaims/Dockerfile:11]:
```suggestion
# Use this for debug, includes curl etc:
# FROM openjdk:8

# Use this for a small production image
FROM openjdk:8-jre-slim
```

File: EventEnabledInsurance/ProjectionClaims/deploy.sh [hunk 2]
```diff
@@ -76,19 +76,7 @@ spec:
...
-              port: 8080
-              scheme: HTTP
-          livenessProbe:
-            httpGet:
-              path: /
-              port: 8080
-              scheme: HTTP
-            initialDelaySeconds: 15
-            periodSeconds: 10
+
```
Comment (DanRoseus) [inline EventEnabledInsurance/ProjectionClaims/deploy.sh:79]: Test if the liveness/readiness probes work now and re-enable

File: EventEnabledInsurance/ProjectionClaims/src/main/java/com/ibm/cp4i/demos/eei/projectionclaims/MyHttpHandler.java [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+            // get a particular record
+            else if (!requestParamValue.isEmpty()) {
+                System.out.println("Requested for a particular quote id: " + requestParamValue);
+                JSONObject trimmedRow = new JSONObject();
+                byte[] byteResponse;
+                try {
+                    JsonNode row = this.monitor.getRow(requestParamValue);
+                    if (row != null) {
+                        trimmedRow.put("quoteid", row.get("quoteid").toString().replace("\"", ""));
+                        trimmedRow.put("name", row.get("name").toString().replace("\"", ""));
```
Comment (DanRoseus) [inline EventEnabledInsurance/ProjectionClaims/src/main/java/com/ibm/cp4i/demos/eei/projectionclaims/MyHttpHandler.java:140]: Why is this removing double quotes? I guess the original data could have double quotes? Or is `toString` returning leading/trailing quotes? If the original data has double quotes in it then we want them to be preserved and returned in the json.

**Review by DanRoseus — CHANGES_REQUESTED** [review]

File: EventEnabledInsurance/ProjectionClaims/src/main/java/com/ibm/cp4i/demos/eei/projectionclaims/MyHttpHandler.java (hunk 3 above)
Comment (Kinshuk1993) [reply EventEnabledInsurance/ProjectionClaims/src/main/java/com/ibm/cp4i/demos/eei/projectionclaims/MyHttpHandler.java:140]: If the replace is removed, the JSON returned looks like:
```
{
    "usstate": "\"VA\"",
    "address": "\"790 Arrowhead Court, Portsmouth\"",
    "claimstatus": "2",
    "name": "\"Ronny Doyle\"",
    "licenseplate": "\"WMC-9628\"",
    "quoteid": "183",
    "email": "\"RonnyDoyle@mail.com\""
}
```

File: EventEnabledInsurance/ProjectionClaims/Dockerfile (hunk 1 above)
Comment (Kinshuk1993) [reply EventEnabledInsurance/ProjectionClaims/Dockerfile]: Done

Comment (Kinshuk1993) [reply EventEnabledInsurance/ProjectionClaims/Dockerfile]: Done

File: EventEnabledInsurance/ProjectionClaims/src/main/java/com/ibm/cp4i/demos/eei/projectionclaims/MyHttpHandler.java (hunk 3 above)
Comment (Kinshuk1993) [reply EventEnabledInsurance/ProjectionClaims/src/main/java/com/ibm/cp4i/demos/eei/projectionclaims/MyHttpHandler.java]: Done

**Review by laurensks — APPROVED** [review]
Looks good :D

**Pull request merged**